### PR TITLE
fix: rename recognize_song

### LINKED ISSUE #20: recognize_song is deprecated
Hi, I use this cli from time to time, can you fix the work with the audio segment and not use deprecated recognize_song?
https://github.com/woodruffw/shaq/blob/58b0cac4fe047d58e493e9a7870fbd0ad9904a95/shaq/_cli.py#L114

An example of how to transfer bytes from an AudioSegment is described here:
https://github.com/shazamio/ShazamIO/issues/103#issuecomment-2081220779

### PR DESCRIPTION
**Pull request by jgafnea**
Fixes #20

### CODE REVIEW

**Comment by woodruffw** [comment]
Thanks for the patch! If I'm reading the release notes correctly, I believe this means we should also update the `shazamio` version range to `>= 0.5`, right? If so, could you change `pyproject.toml` to reflect that as well?

(It looks like `0.6` also includes a bunch of data model fixes, so maybe we should pin `>= 0.6`?)

**Comment by jgafnea** [comment]
Okay, new commit has it pinned `>= 0.6.0`.

File: pyproject.toml [hunk 1]
```diff
@@ -22,7 +22,7 @@ dependencies = [
     "pyaudio ~= 0.2.13",
     "pydub ~= 0.25.1",
     "rich ~= 13.4",
-    "shazamio >= 0.4.0.1,< 0.5.2.0",
+    "shazamio >= 0.6.0",
```
Comment (woodruffw) [inline pyproject.toml:25]: Nitpick: compatible-release constraint instead:

```suggestion
    "shazamio ~= 0.6.0",
```

**Review by woodruffw — APPROVED** [review]
Thanks @jgafnea, LGTM! I'll merge in a bit.

**Comment by woodruffw** [comment]
Released with 0.0.3.

**Pull request merged**